p4include/psa.p4

```diff
@@ -248,6 +223,36 @@ extern PacketReplicationEngine {
   */
 }
 
+extern Clone {
+  /// create a copy of the packet to the specified mirror session.
+  ///
+  /// The PSA specifies four types of cloning, with the packet sourced
+  /// from different points in the pipeline and sent back to ingress
+  /// or to the buffering queue in the egress (@see CloneMethod_t).
+  ///
+  /// @param clone_method  The type of cloning.
+  /// @param session_id    Port mirror session id.
```

Comment:
Parse error: "Port mirror session id". What do you mean by that?
And BTW, you need to also either pass a port or discuss how to set the port for the clone.